bodhi/server/scripts/approve_testing.py

```diff
@@ -128,6 +128,9 @@ def main(argv=sys.argv):
                                 f"These builds {builds_str} have a more recent "
                                 f"build in koji's {update.release.stable_tag} tag.",
                                 author="bodhi")
+                            update.status = UpdateStatus.pending
+                            update.request = None
+                            update.remove_tag(update.release.get_testing_side_tag(update.from_tag))
```

Comment:
You need to create a new tag, if the update is switched back to pending?

Reply:
We should not need,`remove-tag` only remove the tag from all the builds in an update. We need to do that in the case that the update is edited and a build is removed that way the removed build will not have the `testing_side_tag` tag. We can then reuse the original tag.